actions/campaigns.go

```diff
@@ -575,3 +575,41 @@ func GetCampaignBounces(c *gin.Context) {
 		"message": "Campaign bounces not found",
 	})
 }
+
+func DeleteScheduledCampaign(c *gin.Context) {
+	user := middleware.GetUser(c)
+
+	id, err := strconv.ParseInt(c.Param("id"), 10, 64)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"message": "Id must be an integer.",
+		})
+		return
+	}
+	campaign, err := storage.GetCampaign(c, id, user.ID)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"message": "Campaign not found.",
+		})
+		return
+	}
+	if campaign.ID != id {
```

Comment:
Do we really need to check this id?

Reply:
@gud\ Yes because we are deleting only with campaign_id (user_id is not included in scheduled_campaign)

Reply:
yes you are right we do not need it.

Reply:
@Dzalevski you are fetching the campaign with this id why we need to check it again